### DISCUSSION TITLE
is decouple processor actually doing what it is promissing?

### DISCUSSION THREAD

**Discussion by enuret** [open-telemetry/opentelemetry-lambda#1170] [Q&A]
hi! 
I expected that [decouple](https://github.com/open-telemetry/opentelemetry-lambda/tree/main/collector/processor/decoupleprocessor) processor allows lambda to be finished before it ends the exporting.
 
So I made a test with injecting 500ms response delay on collector gateway side and got lambda latencies exactly + 500 ms, so it doesn't seem that it impacts anything in case of slow server response. I saw it is in a development state, did someone already manage to make it work?

**Comment by tylerbenson** [comment]
Did you add the processor in your configuration? The latest layer release should add it automatically now.
